WWDC/BookmarkViewController.swift

```diff
@@ -12,7 +12,7 @@ import PlayerUI
 
 private final class WWDCTextView: NSTextView {
     
-    var textDidChangeHandler: ((_ text: String) -> Void)?
+    var textDidChangeHandler: ((String) -> Void)?
```

Comment:
(Code style disagreement)

I like to add param names to closures so it is very clear what the parameter is, even though it is not necessary I think it's nice to have.